Makefile

```diff
@@ -42,7 +42,7 @@ build-debug:
 {%-   assign build_target = t[0] -%}
 {%-   assign target_type = t[2] -%}
 {%-   if target_type == "debug" -%}
-	# make build-{{build_target}}-{{target_type}}
```

Comment:
@marechal-p would a tab character after the '#' would work in Windows?

Reply:
Yes it will work as long as the `#` is the first character of the line. It is a good idea, looks better.